website/source/docs/internals/telemetry.html.md

```diff
@@ -296,6 +304,18 @@ Thes operations take a request and response with an associated lease and registe
 
 **[S]** Summary (Milliseconds): Time taken to store an updated token entry without writing to the secondary index
 
+### vault_wal_deletewals
+
+**[S]** Summary (Milliseconds): Time taken to delete a Write Ahead Log (WAL)
+
+### vault_wal_persistwals
+
+**[S]** Summary (Milliseconds): Time taken to persist a Write Ahead Log (WAL)
+
+### vault_wal_flushready
+
+**[S]** Summary (Milliseconds): Time taken to flush a ready Write Ahead Log (WAL) to storage
```

Comment:
While you're adding these can you also add `vault_wal_gc_deleted` and `vault_wal_gc_total`, which are the number of wals deleted each GC and the total amount of WALs that can be GC'd

Reply:
Done!